README.md

```diff
@@ -0,0 +1,31 @@
+# prplMesh Agent
+The prplMesh Agent implements a Multi-AP v1.0 certifiable agent (work in progress).
+
+# Build
+Build system is CMAKE, and supports RDKB and UGW cross compilation builds.
+Building natively (Linux only) is partially supported only for development process. Some of the components are skipped / built as stubs in this mode.
```

Comment:
We should also decide on a coding style for .md files. Something that works for me is to have one line per sentence, and not split long lines. Line splitting is anyway done by the editor and MD formatter, so not really needed. One sentence per line has the advantage of keeping diffs small and to the point. The only disadvantage is that with long sentences, the diff becomes hard to read with a tool that doesn't do word diff.

Reply:
Makes sense, I will change the README to have one line per sentence.